### PR TITLE
AAP-31307 Add attributes for title URLs

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Create attributes for docs links so that external links to other docs in the AAP docset don't change when titles are updated.
Backport to 2.5

### CODE REVIEW

File: downstream/attributes/attributes.adoc [hunk 1]
```diff
@@ -277,94 +277,124 @@
...
 :TitleTroubleshootingAAP: Troubleshooting Ansible Automation Platform
+:LinkTroubleshootingAAP: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/troubleshooting_ansible_automation_platform
 //
 // titles/aap-plugin-rhdh-install
 :TitlePluginRHDHInstall: Installing Ansible plug-ins for Red Hat Developer Hub
+:LinkPluginRHDHInstall: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/installing_ansible_plug-ins_for_red_hat_developer_hub
 //
 // titles/aap-plugin-rhdh-using
 :TitlePluginRHDHUsing: Using Ansible plug-ins for Red Hat Developer Hub
+:LinkPluginRHDHUsing: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/using_ansible_plug-ins_for_red_hat_developer_hub
```
Comment (ariordan-redhat) [inline downstream/attributes/attributes.adoc:292]: The hyphen is correct: I checked it against the [2.4 doc on the portal](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html/installing_ansible_plug-ins_for_red_hat_developer_hub/index)

File: downstream/attributes/attributes.adoc [hunk 2]
```diff
@@ -277,94 +277,154 @@
...
+:URLAnalyticsJobExplorer: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/using_automation_analytics
+:LinkAnalyticsJobExplorer: {URLAnalyticsJobExplorer}[{TitleAnalyticsJobExplorer}]
 //
 // titles/develop-automation-content
 :TitleDevelopAutomationContent: Developing automation content
+:URLDevelopAutomationContent: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/developing_automation_content
+:LinkDevelopAutomationContent: {URLDevelopAutomationContent}[{TitleDevelopAutomationContent}]
 //
 // titles/topologies
 :TitleTopologies: Tested deployment models for Ansible Automation Platform
```
Comment (michellemacrh) [inline downstream/attributes/attributes.adoc:428]: I'm updating this today to "Tested deployment models". I can update this also depending on when it's merged 👍

File: downstream/attributes/attributes.adoc [hunk 3]
```diff
@@ -277,94 +277,154 @@
...
+:LinkUpgrade: {URLUpgrade}[{TitleUpgrade}]
 //
 // titles/aap-operator-installation
-:TitleOperatorInstallation: Installing on Openshift Container Platform
+:TitleOperatorInstallation: Installing on OpenShift Container Platform
+:URLOperatorInstallation: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/installing_on_openshift_container_platform
+:LinkOperatorInstallation: {URLOperatorInstallation}[{TitleOperatorInstallation}]
 //
 // titles/aap-installation-guide
 :TitleInstallationGuide: Installing on virtual machines
```
Comment (michellemacrh) [inline downstream/attributes/attributes.adoc:318]: I also think this is due to change to "RPM installation" or similar.

File: downstream/attributes/attributes.adoc (hunk 2 above)
Comment (ariordan-redhat) [reply downstream/attributes/attributes.adoc:428]: Rebased my PR against main to bring in this update.

**Review by g-murray — COMMENTED** [review]
LGTM

**Comment by michellemacrh** [comment]
LGTM

**Pull request merged**